hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestRatisPipelineCreateAndDestory.java

```diff
@@ -97,8 +97,11 @@ public void testPipelineCreationOnNodeRestart() throws Exception {
     }
 
     // try creating another pipeline now
+    RatisPipelineProvider ratisPipelineProvider = (RatisPipelineProvider)
+        pipelineManager.getPipelineFactory().getProvider(
+            HddsProtos.ReplicationType.RATIS);
     try {
-      RatisPipelineUtils.createPipeline(pipelines.get(0), conf);
+      ratisPipelineProvider.createPipeline(pipelines.get(0));
```

Comment:
We can use pipelineManager#create call instead.

Reply:
Done.